randovania/games/planets_zebeth/layout/planets_zebeth_cosmetic_patches.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+import dataclasses
+from enum import Enum
+
+from randovania.games.game import RandovaniaGame
+from randovania.layout.base.cosmetic_patches import BaseCosmeticPatches
+from randovania.lib import enum_lib
+
+
+class PlanetsZebethRoomGuiType(Enum):
+    """Types of Room Name GUI to display."""
+
+    long_name: str
+
+    NONE = "NEVER"
+    WITH_FADE = "WITH_FADE"
+    ALWAYS = "ALWAYS"
+
+
+enum_lib.add_long_name(
+    PlanetsZebethRoomGuiType,
+    {
+        PlanetsZebethRoomGuiType.NONE: "Never",
+        PlanetsZebethRoomGuiType.WITH_FADE: "When entering a room",
+        PlanetsZebethRoomGuiType.ALWAYS: "Always",
+    },
+)
+
+
+@dataclasses.dataclass(frozen=True)
+class PlanetsZebethCosmeticPatches(BaseCosmeticPatches):
+    show_unexplored_map: bool = True
```

Comment:
i'm curious what your reasoning is for making this an option.

Reply:
show_unexplored_map aka enable map from start for other games

Reply:
I can read that, yes. My question was, why is this an option, instead of forced?

Reply:
why would it be forced when other games like Prime 1 and 2 give us an option?

Reply:
I completely forgot about this, whoops. Nevermind.